convert2rhel/pkghandler.py

```diff
@@ -293,12 +292,14 @@ def print_pkg_info(pkgs):
     (https://bugzilla.redhat.com/show_bug.cgi?id=1876561).
     """
     max_nvra_length = max(map(len, [get_pkg_nvra(pkg) for pkg in pkgs]))
-    max_packager_length = max(max(map(len, [get_packager(pkg) for pkg in pkgs])), len("Packager"))
+    max_packager_length = max(
+        max(map(len, [get_vendor(pkg) if hasattr(pkg, "vendor") else get_packager(pkg) for pkg in pkgs])),
+        len("Vendor/Packager"))
```

Comment:
Nit:
It is not that readable, here you could find some concepts to improve the readability:
```python
max(itertools.chain((n*"*" for n in range(5)), ("******",)), key=len)
```